### ISSUE TITLE
False positive for chainable on minBy

### ISSUE THREAD

**Issue by corbinu** [wix/eslint-plugin-lodash#37]
I am getting a false positive for chainable

I am starting with an array like this:
```
let test = [
    {
        "name": "a",
        "age": 18
    },
    {
        "name": "b",
        "age": 21
    },
    {
        "name": "a",
        "age": 55
    }
];
```

I want to convert this to the min age for each name
Correct end result is
```
[{"name": "a", "age": 18},{"name": "b", "age": 21}]
```

I am trying this:
```
testResult = _(test)
    .groupBy("name") // {"a": [{"name": "a", "age": 18}, {"name": "a", "age": 55}], "b": [{"name": "b", "age": 21}]}
    .map((sameName) => _.minBy(sameName, "age")) // [{"name": "a", "age": 18}, {"name": "b", "age": 21}]
    .value();

```
OR
```
let groupedNames = _.groupBy(test, "name");
testResult = _.map(groupedNames, (sameName) => _.minBy(sameName, "age"));
```

I am not sure why I get an error on both of these of "refer lodash chain (lodash/prefer-lodash-chain)"
This seems to be incorrect as minBy is not chainable

Perhaps there is a better way though?

**Comment by ganimomer** [comment]
Hi,
This is generally what I would do for this problem (either is fine, though I prefer chaining, since it doesn't require creating intermediate variables).
This looks like it really is a false positive, we'll look into it.
